ngrinder-controller/src/main/java/org/ngrinder/infra/config/DynamicCacheConfig.java

```diff
@@ -149,6 +149,7 @@ public CacheConfigHolder cacheConfigMap() {
 		cm.addDistMap(DIST_MAP_NAME_SAMPLING, 15);
 		cm.addDistMap(DIST_MAP_NAME_MONITORING, 15);
 		cm.addDistMap(DIST_MAP_NAME_AGENT, 10);
+		cm.addDistMap(DIST_MAP_CACHE_RECENTLY_USED_AGENTS, 1 * DAY);
```

Comment:
Could u rename all const here? 
I think it's not consistent b/w local and remote cache...

For example, I think the following const is better.

* DIST_CACHE_SAMPLING
* DIST_CACHE_RECENTLY_USED_AGENTS
* LOCAL_CACHE_USERS

Please fix them by seperate commit.